src/dispatcher/server.rs

```diff
@@ -1,25 +1,28 @@
+#[macro_use]
```

Comment:
Now, in 2018 this example will not compile:

```rust
use log::error;

fn main() {
    error!("error message");
}
```
will give an error message about not finding the __impl_log! macro. This is because unlike in the 2015 edition, macros are namespaced and we must import them. We could do

`use log::{__impl_log, error};`
which would make our code compile, but `__impl_log` is meant to be an implementation detail!

Reply:
that's from the rust docs...
I like the old syntax better because it is short and has no weird edge cases...

Reply:
:+1: